src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -17,7 +17,12 @@
  *
  * @author Kohsuke Kawaguchi
  */
+@SuppressWarnings({ "UnusedDeclaration" })
 public class GHOrganization extends GHPerson {
+
+    private boolean has_organization_projects;
```

Comment:
Please comment on the field naming. Snake case isn't common, so I suspect we need it for de-serialization from JSON.